### PR TITLE
Feature/imgui

### PR DESCRIPTION
**Pull request by Alekssasho**

### CODE REVIEW

**Comment by Alekssasho** [comment]
Failing build is fixed in @NDobrev branch and will be merged separately

File: include/Zmey/Graphics/Renderer.h [hunk 1]
```diff
@@ -13,6 +13,8 @@
 #include <Zmey/Graphics/Managers/UploadHeap.h>
 #include <stdint.h>
 
+struct ImDrawData;
```
Comment (NDobrev) [inline include/Zmey/Graphics/Renderer.h]: remove(not used)

File: include/Zmey/Graphics/Backend/Vulkan/VulkanDevice.h [hunk 2]
```diff
@@ -88,7 +90,8 @@ class VulkanDevice : public Device
 	VkFormat m_SwapChainImageFormat;
 	VkExtent2D m_SwapChainExtent;
 
-	VulkanCommandPoolHandle m_CommandPool{ VulkanCommandPoolDeleter{&m_Device} };
+	VulkanCommandPoolHandle m_CommandPool{ VulkanCommandPoolDeleter{ &m_Device } };
+	VulkanCommandPoolHandle m_UICommandPool{ VulkanCommandPoolDeleter{&m_Device} };
```
Comment (NDobrev) [inline include/Zmey/Graphics/Backend/Vulkan/VulkanDevice.h]: space after { and before }

File: include/Zmey/Graphics/Renderer.h (hunk 1 above)
Comment (Alekssasho) [reply include/Zmey/Graphics/Renderer.h]: fxd

File: include/Zmey/Graphics/Backend/Vulkan/VulkanDevice.h (hunk 2 above)
Comment (Alekssasho) [reply include/Zmey/Graphics/Backend/Vulkan/VulkanDevice.h]: fxd

**Pull request merged**